src/BassetManager.php

```diff
@@ -30,6 +36,10 @@ public function __construct()
         $this->cachebusting = $cachebusting ? (string) Str::of($cachebusting)->start('?') : '';
         $this->basePath = (string) Str::of(config('backpack.basset.path'))->finish('/');
 
+        $this->cacheMap = new CacheMap();
+        $this->loading = new LoadingTime();
```

Comment:
Hmm I found this a bit weird... "loading" is not a noun so it's weird to do stuff to it, right? 😀

```suggestion
        $this->loader = new LoadingTime();
```